.github/workflows/build-docker.yaml

```diff
@@ -1,48 +1,58 @@
-name: Build scpca-tools image
+name: Build scpca-tools images
 
 on:
   push:
     branches:
       - main
     tags:
-      - 'v*'
+      - "v*"
   pull_request:
     branches:
       - main
 
 env:
   REGISTRY: ghcr.io
-  IMAGE_NAME: alexslemonade/scpca-tools
+  IMAGE_NAME: alexslemonade/scpcatools
 
 jobs:
   build_and_push:
     runs-on: ubuntu-22.04
     permissions:
       contents: read
       packages: write
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - dockerfile: Dockerfile
+            suffix: ""
+          - dockerfile: scpcatools_slim.Dockerfile
```

Comment:
Don't we also want to build any other images here too? So we would need the zellkonverter/ AnnData image?

Reply:
Oh we would... I just wanted to start with two for testing.

I also have a thought that we might be able to build off the other images off the slim image, which could save time, but I haven't tested how that would work quite yet.